src/components/icons/ReversibleArrows.tsx

```diff
@@ -4,23 +4,27 @@ const ReversibleArrows: React.FC = () => {
     return (
```

Comment:
this changes the rate constants to cursive `k`s, which is what they're supposed to be